components/ResultsPage/YourAnswers.tsx

```diff
@@ -0,0 +1,275 @@
+import { Link as DSLink } from '@dts-stn/decd-design-system'
+import { useRouter } from 'next/router'
+import { useStore, useTranslation } from '../Hooks'
+import { WebTranslations } from '../../i18n/web'
+import { livingCountry as enCountry } from '../../i18n/api/countries/en'
+import { livingCountry as frCountry } from '../../i18n/api/countries/fr'
+import en from '../../i18n/api/en'
+import fr from '../../i18n/api/fr'
+import { Locale } from '../../utils/api/definitions/enums'
+import { FieldKey } from '../../utils/api/definitions/fields'
+
+
+
+export const YourAnswers: React.VFC<{
+  title: string
+  questions: Array<[ string, string ]>
+}> = ({ title, questions }) => {
+  const root = useStore()
+
+  const tsln = useTranslation<WebTranslations>()
+  
+  const currentLocale = useRouter().locale
+  const locale = currentLocale == 'en' ? Locale.EN : Locale.FR
+
+  const answers = questions.filter( question => question[0] !== '_language')
+  const answersKeys = answers.map(([key,_]) => key)
+
+  function getLivingCountry(country: string): {key: string,text: string} {
```

Comment:
small thing, but it makes me think your local env isn't set up properly - you have `{key: string,text: string}` - there is no space after the comma. Git should be running Prettier on every commit to prevent inconsistent syntax from getting into the repo. If your commit hook is not running, try using `yarn run prepare` to install the `husky` hook.

If you run Prettier on your code (using built-in vscode function, or `yarn run reformat`), you should see a lot of changes in the formatting!